ProjectSettings/QualitySettings.asset

```diff
@@ -187,7 +187,7 @@ QualitySettings:
     Web: 5
     WebGL: 3
     WiiU: 5
-    Windows Store Apps: 0
+    Windows Store Apps: 5
```

Comment:
Please revert this file.

Reply:
Was unsure why this changed, possibly unity it'self. But will revert this one